db/schema.rb

```diff
@@ -27,94 +27,73 @@
 
   add_index "issue_assignments", ["delivered"], name: "index_issue_assignments_on_delivered", using: :btree
 
-  create_table "issues", force: true do |t|
+  create_table "issues", force: :cascade do |t|
     t.integer  "comment_count"
-    t.string   "url",             limit: nil
-    t.string   "repo_name",       limit: nil
-    t.string   "user_name",       limit: nil
+    t.string   "url"
+    t.string   "repo_name"
+    t.string   "user_name"
     t.datetime "last_touched_at"
     t.integer  "number"
-    t.datetime "created_at",                                  null: false
-    t.datetime "updated_at",                                  null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
     t.integer  "repo_id"
-    t.string   "title",           limit: nil
-    t.string   "html_url",        limit: nil
-    t.string   "state",           limit: nil
-    t.boolean  "pr_attached",                 default: false
+    t.string   "title"
+    t.string   "html_url"
+    t.string   "state"
+    t.boolean  "pr_attached",     default: false
+    t.string   "creator"
   end
 
-  create_table "opro_auth_grants", force: true do |t|
-    t.string   "code",                    limit: nil
-    t.string   "access_token",            limit: nil
-    t.string   "refresh_token",           limit: nil
-    t.text     "permissions"
-    t.datetime "access_token_expires_at"
-    t.integer  "user_id"
-    t.integer  "application_id"
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
-  end
-
-  create_table "opro_client_apps", force: true do |t|
-    t.string   "name",        limit: nil
-    t.string   "app_id",      limit: nil
-    t.string   "app_secret",  limit: nil
-    t.text     "permissions"
-    t.integer  "user_id"
-    t.datetime "created_at",              null: false
-    t.datetime "updated_at",              null: false
-  end
-
```

Comment:
These two tables `opro_auth_grants` and `opro_client_apps` were added in the first commit referenced above. They don't seem to have corresponding migrations. I'll add them back in manually.

This is just for my edification, but isn't changing the schema directly like this generally not a good idea? I'm happy to restore it, but won't we have this issue again any time anyone needs to make a migration? I'm just trying to understand why these changes seem to have been made directly. Does it have anything to do with these tables corresponding to POROs?